### PR TITLE
add cni configure example

### PR DESCRIPTION
**Pull request by heartlock**

### CODE REVIEW

**Comment by heartlock** [comment]
@resouer @feiskyer

File: README.md [hunk 1]
```diff
@@ -42,6 +42,43 @@ EOF
 systemctl restart hyperd
 ```
 
+Setup CNI networking
+
+Follow the steps below in order to setup networking in your pods using the CNI
```
Comment (feiskyer) [inline README.md]: Please merge those two into one short line (so as to in same style as other places)

**Comment by heartlock** [comment]
updated @feiskyer

**Comment by feiskyer** [comment]
LGTM. Thanks

**Pull request merged**